packages/wp-story-editor/src/components/documentPane/publish/publish.js

```diff
@@ -366,7 +363,7 @@ function PublishPanel({ nameOverride }) {
                 ariaLabel={__('Poster image', 'web-stories')}
                 onChangeErrorText={posterErrorMessage}
                 imgProps={featuredMedia}
-                canUpload={hasUploadMediaAction || enablePosterHotlinking}
+                canUpload={hasUploadMediaAction}
```

Comment:
Isn't that a change in semantics?

Previously: `canUpload={hasUploadMediaAction || enablePosterHotlinking}`

But now we allow poster hotlinking by default (basically making `enablePosterHotlinking` always `true`), so technically this should be `canUpload={true}`, no?

Reply:
Good point -- updated.